test-network/prometheus-grafana/README.md

```diff
@@ -0,0 +1,47 @@
+# Description
+
+This sample provides an environment to display and capture metrics from the test-network in real time. It consists of a docker-compose file that starts a prometheus and grafana server setup configured to collect and display metrics for the test network.
+
+# Requirements
+
+This sample has been tested and is meant to be used on **linux** in order to fully benefit from its capabilities.
+You will need to have installed **docker-compose with version 1.29 or above** (note that this is higher than the v1.14 requirement requested for the test-network).
+
+# How to use
+
+1. Bring up the test-network -> **./network.sh up**
```

Comment:
I would be a bit more descriptive. Go to the `test-network` directory and run ...
I would also run some thing more interesting, like `./network up createChannel -ca` so that folks see more metrics.